src/cli/project/platform/remove.rs

```diff
@@ -38,7 +38,7 @@ pub async fn execute(mut project: Project, args: Args) -> miette::Result<()> {
     // Remove the platform(s) from the manifest
     project
         .manifest
-        .remove_platforms(&platforms, &feature_name)?;
+        .remove_platforms(platforms.clone(), &feature_name)?;
```

Comment:
Current (or previous) implementation does not notify the user is a platform cannot be found for the feature.

Is this desired? this seems like a different behaviour than for dependency removal, which will Err if the dependency is not found.

Reply:
Yeah that seems wrong, not removing without an error can lead to users thinking it removed something but they had a typo. Resulting in unexpected behavior.

Reply:
OK. I created an issue out of that, that can be implemented in a separate PR.